tests/acceptance/features/bootstrap/WebDav.php

```diff
@@ -2925,6 +2929,65 @@ public function userHasDeletedEverythingInFolder($user, $folder) {
 		$this->userDeletesEverythingInFolder($user, $folder, true);
 	}
 
+	/**
+	 * @When user :user downloads the preview of :path with width :width and height :height using the WebDAV API
+	 *
+	 * @param $user
+	 * @param $path
+	 * @param $width
+	 * @param $height
+	 *
+	 * @return void
+	 */
+	public function downloadPreview($user, $path, $width, $height) {
+		$urlParameter = [
+			'x' => $width,
+			'y' => $height,
+			'forceIcon' => '0',
+			'preview' => '1'
+		];
+		$this->response = $this->makeDavRequest(
+			$user, "GET", $path, [], null, "files", 2, false, null, $urlParameter
+		);
```

Comment:
```suggestion
		$this->downloadPreviewOfOtherUser($user, $path, $user, $width, $height);
```
IMO this will work and avoid duplicating the `$urlParameter` variable setting.